### PR TITLE
Caffe2 8ce4e2e

### PR DESCRIPTION
**Pull request by gppezzi**
requires #162

### CODE REVIEW

File: easybuild/easyconfigs/c/Caffe/Caffe-1.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    'files': ['bin/caffe'] +
+             ['lib64/%s' % x for x in ['libcaffe.so', 'libproto.a']] +
+             ['python/caffe/_caffe.so'] +
+             ['include/caffe/proto/caffe.pb.h'],
+    'dirs': [],
+}
+
+#sanity_check_commands = [('python', "-c 'import caffe'")]
+
+moduleclass = 'devel'
```
Comment (jgphpc) [inline easybuild/easyconfigs/c/Caffe/Caffe-1.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb]: already merged in 
https://github.com/eth-cscs/production/blob/6e84c1faa19d9bd2c45c565c1b7e1403d9aa7258/easybuild/easyconfigs/c/Caffe/Caffe-1.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb   
?

File: easybuild/easyconfigs/c/Caffe2/Caffe2-0.7.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+description = """
+Caffe is a deep learning framework made with expression, speed,
+and modularity in mind. It is developed by the Berkeley Vision
+and Learning Center (BVLC) and community contributors.
+"""
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
+toolchainopts = {'optarch': True, 'pic': False, 'verbose': False}
+
+#sources = ['v%(version)s.tar.gz']
```
Comment (jgphpc) [inline easybuild/easyconfigs/c/Caffe2/Caffe2-0.7.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb]: remove \# ?

File: easybuild/easyconfigs/c/Caffe2/Caffe2-0.7.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    'files': ['lib/libCaffe2_CPU.so'] +
+             ['lib/libCaffe2_GPU.so'] +
+             ['include/caffe2/proto/caffe2.pb.h'],
+    'dirs': ['binaries'],
+}
+
+#sanity_check_commands = [('python', "-c 'from caffe2.python import core'")]
+
+moduleclass = 'devel'
+
```
Comment (jgphpc) [inline easybuild/easyconfigs/c/Caffe2/Caffe2-0.7.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb]: rm lastline

**Comment by gppezzi** [comment]
Thanks for the review @jgphpc

**Pull request merged**